src/app/components/Footer/index.tsx

```diff
@@ -72,53 +72,67 @@ const SocialIconMessage: React.FunctionComponent<{
   </Styled.SocialIcon>
 );
 
+function LinkList({children}: React.PropsWithChildren<{}>) {
+  return (
+    <Styled.LinkListWrapper>
+      {React.Children.toArray(children).map((c) => <li>{c}</li>)}
```

Comment:
I noticed this one doesn't set the key property, unlike the other PR

Reply:
Oh, yeah. No warning on the Rex system for that. The set of children won't be changing, but I pushed a use-index-as-key update.